Realm/Realm/Attributes/GeneratedAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ////////////////////////////////////////////////////////////////////////////
+// //
+// // Copyright 2022 Realm Inc.
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License")
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// // http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+// //
+// ////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.ComponentModel;
+
+namespace Realms
+{
+
+    //TODO Add documentation
```

Comment:
I don't think we should document this - instead, let's hide it from the docs, as users are not supposed to interact with it.